woocommerce/woocommerce-android
# Add toolbar for ShippingLabelAddressSuggestionFragment

Part of #10570 

### Description

This PR introduces individual toolbars for ShippingLabelAddressSuggestionFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Create a shipping label CTA -> Ship from -> Address suggestion validation) screen and ensure the toolbar and its action work as expected on both phone and tablet.
